science/ngspice/files/patch-fix-linking.diff

```diff
@@ -0,0 +1,12 @@
+--- src/Makefile.in	2024-07-13 01:09:10.000000000 +0800
++++ src/Makefile.in	2024-08-31 13:25:22.000000000 +0800
+@@ -875,7 +875,8 @@
+ 	-e 's|@XSPICEINIT[@]|$(XSPICEINIT)|g' \
+ 	-e 's|@pkglibdir[@]|$(spinitpath)|g'
+ 
+-AM_CFLAGS = -static
++# MacOS linker does not need this.
++# AM_CFLAGS = -static
```

Comment:
I think this might be a weird old leftover, ngspice goes all the way back to SPICE3 essentially.